src/components/ErrorBoundary/index.tsx

```diff
@@ -4,11 +4,17 @@ import BugsnagPluginReact from "@bugsnag/plugin-react";
 import { environmentalVariables } from "utils";
 import ErrorFallback from "./ErrorFallback";
 
-const { getBugsnagApiKey, isProduction } = environmentalVariables;
+const {
+  getBugsnagApiKey,
+  isProduction,
+  getAppVersion,
+  getReleaseStage,
+} = environmentalVariables;
+
 let bugsnagStarted = false;
 
 // This error boundary is used during local development
-class DefaultErrorBoundary extends React.Component {
+class DefaultErrorBoundary extends React.Component<{}, { hasError: boolean }> {
```

Comment:
What's the purpose of the initial empty object?

Reply:
The initial object is the typing for the component props. Since i did not want to do anything with them i had to pass an empty type.  The second value is state and you have to pass in the props type before passing in the state type. It does not let me use an undefined value.

Reply:
Thanks, I've never really used class components with TypeScript! Cool 😎